.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"  
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "pip" # See documentation for possible values
```

Comment:
Would it be possible to add it for nuget (c# package manager) too?

Reply:
I'll leave that for you :p